README.md

```diff
@@ -4,6 +4,61 @@
 
 This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [developer docs](https://azure.github.io/azure-sdk-for-cpp).
 
+## Getting started
+
+The Azure SDK for C++ is compatible with a number of different development environments and tools. The following instructions will utilize [Visual Studio](https://visualstudio.microsoft.com/) or [VSCode](https://code.visualstudio.com/) as the IDE, [CMake](https://cmake.org/) for build automation, and [vcpkg](https://vcpkg.io/) as our package manager.
+
```

Comment:
For VS code, do you need Ninja as a build orchestration tool, or will the Visual Studio generator work?

Reply:
The vcpkg docs don't call it out specifically so I don't think so.

Reply:
Unless it's hidden somewhere in the C++ language or CMake extensions I don't believe it's needed.